time/src/main/java/net/jqwik/api/time/YearMonthArbitrary.java

```diff
@@ -0,0 +1,115 @@
+package net.jqwik.api.time;
+
+import java.time.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+
+import static org.apiguardian.api.API.Status.*;
+
+/**
+ * Fluent interface to configure the generation of year and month values.
+ */
+@API(status = EXPERIMENTAL, since = "1.4.0")
+public interface YearMonthArbitrary extends Arbitrary<YearMonth> {
```

Comment:
I'd get rid of all the greate/less/orEqual methods. Too much API maintenance burden for too little value.